### ISSUE TITLE
How to use auto-resize-font.el?

### ISSUE THREAD

**Issue by LouisAsh** [brandon-rhodes/dot-emacs#2]
Dear Brandon,

The idea you have developed into yours `auto-resize-font.el` is something that I have been trying to achieve in Spacemacs for a long time. I am now trying it in plain Emacs to learn how it works, but I could not get it to function. Would care giving me a quick pointer on how to use it? I included its code in my `.emacs` file but nothing happened. Where exactly should I call the main function defined in your code?

Thanks!

**Comment by brandon-rhodes** [comment]
Good question! I can't remember exactly why I wrote it — maybe I was experimenting with a tiling window manager and wanted it to choose a smaller font when I collapsed Emacs to only being half of the width of the screen? — but it does not seem to work with the most recent version of Emacs, so I won't be using it again any time soon. Feel free to use the code, and to post to this issue later if you get it working!
